### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by schoeberl** [t-crest/patmos#159]
We have following failing tests:
```
The following tests FAILED:
	 84 - mrtc-lms_hw (Timeout)
	114 - mrtc-st_hw (Timeout)
	118 - mrtc-whet_hw (Timeout)
	1006 - torture-920501-2_hw (Timeout)
	1012 - torture-920501-6_hw (Timeout)
	1510 - torture-arith-rand-ll_hw (Timeout)
```

Maybe they just run too long with a 4-core Patmos.

**Comment by Emoun** [comment]
Are these from the benchmark repository? If so, I think you can [change the timeout limit here](https://github.com/t-crest/patmos-benchmarks/blob/4d5a76ab65df952ac41dd2da85fe067a3bb2d96a/cmake/patmos-clang-toolchain.cmake#L328C5-L328C61).

**Comment by schoeberl** [comment]
yes, the standard benchmarks thsat run every night on Helena, if she is not down ;-) I did not try, just a note that is might be just a timeout issue. Or they run forever.

**Comment by Emoun** [comment]
I'm assuming this is not relevant anymore and closing
